wolfcrypt/src/asn.c

```diff
@@ -17556,7 +17556,7 @@ void FreeOcspRequest(OcspRequest* req)
 
 int CompareOcspReqResp(OcspRequest* req, OcspResponse* resp)
 {
-    int cmp;
+    int cmp = 0; /* start as matching if both req and resp have no values */
```

Comment:
Why start assuming a match, especially when there's not a match (no values can't match can they?)?

Reply:
It is a good question, the thought was that if both req and resp have no values to compare then it would be similar to comparing NULL to itself. The current sanity checks would prevent this case though, and it looks like a false positive from the static analysis tool. Having it set to a default of not matching will be safer for future use and code changes. I'll fix it.